### DISCUSSION TITLE
Documentation of GUI functions/buttons

### DISCUSSION THREAD

**Discussion by torbenott** [JingyiGF/HERBS#2] [General]
Hi, 
first of all great tool, thanks! I'm trying to use this tool to reconstruct probe locations from rat histology data. I can load the atlas and images just fine but I find navigating the GUI not entirely intuitive. Is there documentation/function reference somewhere? I looked in the git readme and tutorial but could not find much.
Thanks!

**Comment by JingyiGF** [comment]
Hi,
Thank you message. I fully understand how confusing to use it without User Manual. Believe me, We are working on it. I am currently occupied by the revision of a paper. But we will update it ASAP when we finish it. If you are interested in testing HERBS, you are welcome to email me so that I can show you some tricks :-)

**Comment by JingyiGF** [comment]
Hi, We upload an ongoing CookBook.pdf for HERBS.

**Reply by torbenott** [reply]
thanks!

**Comment by taningh86** [comment]
Hi Jingyi,

Thanks for the great tool. I am trying to upload the allen mouse brain atlas but it's been stuck at 1% for the last few hours (image attached). Any suggestions to reslve this will be very helpful. Thanks!
![image](https://user-images.githubusercontent.com/95593601/197880066-347684b0-37ba-43b7-af22-9216b1705f95.png)

**Reply by JingyiGF** [reply]
Hi @taningh86, thank you for the feedback. this is super wired. I tried it also and could not re-produce the problem. Could you please try again? After click the process button, please do not click other buttons until it finishes. It should not take more than 10 mins from 1% to 10% . Please let me know if the process freeze again.
